src/api/warp.ts

```diff
@@ -152,10 +155,29 @@ function createQueryParamHash(evalOptions: Partial<EvaluationOptions>): string {
   return hash.digest('hex');
 }
 
+function createSortKeyBlockHeightHash({
+  sortKey,
+  blockHeight,
+}: {
+  sortKey: string | undefined;
+  blockHeight: number | undefined;
+}) {
+  const hash = createHash('sha256');
+  hash.update(`${sortKey}-${blockHeight}`);
```

Comment:
missing padded 0's for JS timestamp - not sure if this is require by smartweave base standard but its what warp does.

this is also backwards - its blockheight-timestamp-sortkey.

Or am i misinterpreting the purpose of this?

Reply:
it's our own internal hash for caching - nothing to do with warps sort key hash creation